### PR TITLE
feat(fastly-cdn): Initial observability pack creation

### PR DESCRIPTION
**Pull request by nr-jbiggley**
## Summary

A concise description of the changes being introduced.
New observability pack for Fastly CDN, one of the launch partners for I/O

### Screenshots

fastly-cdn-logs

### CODE REVIEW

File: packs/fastly/config.yml
Comment (polfliet) [inline packs/fastly/config.yml:45]: Did you forgot to remove this?

File: packs/fastly/config.yml
Comment (polfliet) [inline packs/fastly/config.yml:46]: And this one too?

File: packs/fastly/dashboards/fastly-cdn.json
Comment (polfliet) [inline packs/fastly/dashboards/fastly-cdn.json:4]: this `permissions` line will have to be removed manually for now
and all of the `description` fields should be an empty string "" instead of null
(otherwise the schema validation check will fail, these are some known issues we will improve but have to be fixed manually now)
Thanks!

File: packs/fastly/config.yml
Comment (polfliet) [inline packs/fastly/config.yml:20]: Let's separate in 2 lines (authors is a list)

File: packs/fastly/config.yml
Comment (polfliet) [inline packs/fastly/config.yml:36]: @kidk will we have to create an installPlan?

**Review by polfliet — CHANGES_REQUESTED** [review]
Thanks for the contribution @nr-jbiggley! I made a couple of comments

File: packs/fastly/config.yml
Comment (nr-jbiggley) [reply packs/fastly/config.yml]: Done

File: packs/fastly/config.yml
Comment (nr-jbiggley) [reply packs/fastly/config.yml:32]: Removed install plan. Does that work @kidk @polfliet?

File: packs/fastly/config.yml
Comment (nr-jbiggley) [reply packs/fastly/config.yml]: Yep! Cleaned up

File: packs/fastly/dashboards/fastly-cdn.json
Comment (nr-jbiggley) [reply packs/fastly/dashboards/fastly-cdn.json]: Done

**Review by nr-jbiggley — COMMENTED** [review]
Pending review by @polfliet & @kidk

**Review by nr-jbiggley — COMMENTED** [review]
Added reference for support as per @polfliet request via Slack

File: packs/fastly/config.yml
Comment (polfliet) [reply packs/fastly/config.yml:34]: @kidk will create an installPlan for this quickstart. I think it is ok to merge in now and create that installPlan later

Comment (polfliet) [reply packs/fastly/config.yml:34]: actually, without an installPlan the installation experience will not launch and dashboards/alerts will not be created, so let's hold off while Sam adds that.
other than that, looking great!

**Review by polfliet — APPROVED** [review]
Approving since this will already work to list Fastly.
But we need an installPlan to also integrate it with Virtuoso, Sam will add that

**Comment by polfliet** [comment]
@nr-jbiggley can you fix the merge conflict, and then we will approve and merge
Sam will add the installPlan once it is merged
thanks again!

**Comment by nr-jbiggley** [comment]
Created an untenable mess with conflicting PRs. Will try again!

**Pull request closed without merging**
